tests/codeception/common/_support/FixtureHelper.php

```diff
@@ -58,6 +58,30 @@ public function globalFixtures()
     }
 
     /**
+     * Workaround used to load custom fixtures for the test
+     * @link https://github.com/Codeception/Codeception/issues/2258
+     * @link https://github.com/yiisoft/yii2/issues/5575
+     * @param array $fixtureConfig
+     */
+    public function loadFixturesForTest(array $fixtureConfig)
+    {
+        $fixtures = ArrayHelper::merge(
```

Comment:
In this case the line https://github.com/yiisoft/yii2-app-advanced/pull/63/files#diff-99f4c9558f982bc7a6693208b5cdbea8R4 seems to be redundant because we load it's fixture with `$this->fixtures()`.
Seems that I forget to remove it.

> any example would be helpful, can you provide one?

`$this->loadFixtures()` will truncate all tables which has a relevant fixture. So if you have a global fixture for `user` table you have to explicitly load it in `$fixtureConfig` or all this data will be overridden by `$this->fixtures()`